packages/@aws-cdk/aws-cloudwatch/lib/graph.ts

```diff
@@ -510,6 +511,122 @@ export class SingleValueWidget extends ConcreteWidget {
   }
 }
 
+/**
+ * The number of top contributors to show.
+ */
+export enum TopContributors {
+  /**
+   * The top 10 contributors
+   */
+  TOP10 = 10,
+  /**
+   * The top 25 contributors
+   */
+  TOP25 = 25,
+  /**
+   * The top 50 contributors
+   */
+  TOP50 = 50,
+  /**
+   * The top 100 contributors
+   */
+  TOP100 = 100,
+}
+
+/**
+* Statistic to use over the aggregation period
+*/
+export enum OrderStatistic {
+  /**
+   * All values submitted for the matching metric added together.
+   * This statistic can be useful for determining the total volume of a metric.
+   */
+  SUM = 'Sum',
+  /**
+   * The highest value observed during the specified period.
+   * You can use this value to determine high volumes of activity for your application.
+   */
+  MAXIMUM = 'Maximum'
+}
+
+/**
+* Properties for a ContributorInsightsWidget
+*/
+export interface ContributorInsightsWidgetProps extends MetricWidgetProps {
+  /**
+   * Account the contributor insights rule comes from.
+   *
+   * @default None
+   */
+  readonly account?: string;
+  /**
+   * Number of top contributors to display.
+   *
+   * @default - Top 10
```

Comment:
```suggestion
   * @default TopContributors.TOP_10
```

I'm also assuming that we are using an enum here because the only valid values are 10/25/50/100? Is that correct?

Reply:
I think it can be a number. Will test and change